WebRoot/templates/mail/Message.template

```diff
@@ -470,6 +473,34 @@
 			</td>
 		</tr>
 	</table>
+
+	<div class="CompDragArea" id='${compDragAreaId}'>
```

Comment:
I see inconsistency in using the quotes. At some places its double quotes and single quotes at other. Would be good to have same everywhere !